packages/palette/src/elements/Checkbox/Checkbox.tsx

```diff
@@ -1,11 +1,13 @@
 import React from "react"
 import styled, { css } from "styled-components"
 import { color } from "../../helpers"
-import { Clickable, ClickableProps } from "../Clickable"
```

Comment:
impertinent to this PR & non-blocking -- out of curiosity should [the `Clickable` component](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Clickable/Clickable.tsx) specify `type="button"` on its underlying `button` component? I'm not sure how much we're using it, and where, but I think I'd be surprised if I wrapped something in `clickable` and it acted as a `type="submit"` button instead of `type="button"`.

Reply:
Yeah this is a good point/good idea. Will take a look at this separately.